src/test/java/org/opentripplanner/model/plan/legreference/LegReferenceSerializerTest.java

```diff
@@ -0,0 +1,26 @@
+package org.opentripplanner.model.plan.legreference;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.model.FeedScopedId;
+import org.opentripplanner.model.calendar.ServiceDate;
+
+class LegReferenceSerializerTest {
+
+    @Test
+    void testScheduledTransitLegReferenceRoundTrip() {
```

Comment:
I know that this is early days for this feature but would it make sense to store a serialised version of the reference and check that it can be decoded?

If we aren't giving any guarantees about how long these references are valid, then we can skip that.

Reply:
Yes, I can do that.